strings/base_agile_ref.h

```diff
@@ -71,10 +71,12 @@ namespace winrt::impl
 
     using update_module_lock = module_lock_updater<true>;
 
+#ifdef WINRT_REG_FREE
```

Comment:
This ifdef makes it so that there is definitely no LoadLibrary usage when regfree is inactive.